### DISCUSSION TITLE
haproxy support ?

### DISCUSSION THREAD

**Discussion by aderumier** [openappsec/openappsec#242] [Feature Requests]
Hi,

is their any plan to add support for haproxy?  (through spoe protocol for example)

https://www.haproxy.com/fr/blog/extending-haproxy-with-the-stream-processing-offload-engine/

**Comment by orianelou** [comment]
Hi,

HAProxy is a strong candidate to be added to the roadmap but isn't officially planned.
If you would like to contribute to the project by adding support to HAProxy yourself we will be very glad to support you (or any community member that would like to develop it).  Yet we encourage you to reach out (using info@openappsec.io) before a PR.

**Comment by orianelou** [comment]
If you're interested in this feature, please 👍 or upvote this issue to show your support!
